src/main/java/io/vlingo/wire/fdx/inbound/rsocket/RSocketChannelInboundReader.java

```diff
@@ -60,7 +55,7 @@ public void close() {
     if (this.serverSocket != null && !this.serverSocket.isDisposed()) {
       try {
         this.serverSocket.dispose();
-      } catch (final Throwable t) {
+      } catch (final Exception t) {
```

Comment:
Do we really want to propagate an unchecked exception if the socket was not closed?